README.md

```diff
@@ -3,17 +3,21 @@
 Provides support for parsing and emitting Yaml documents.
 
 This package includes the [full libyaml C library version 0.1.5 by Kirill
-Simonov](http://pyyaml.org/wiki/LibYAML) in the package so you
```

Comment:
Changed haddock quoting (`"Text.Libyaml"`) to markdown quoting (``Text.Libyaml``). Also `@aeson@` to ``aeson``. I couldn't find anywhere this was interpreted has haddock rather than markdown where the former would have been appropriate.